pkg/policy/l4.go

```diff
@@ -191,13 +214,27 @@ func (l4 *L4Filter) ToKeys(direction trafficdirection.TrafficDirection) []Key {
 }
 
 // IdentitySelectionUpdated implements CachedSelectionUser interface
+// This call is made while holding selector cache lock, must beware of deadlocking!
 func (l4 *L4Filter) IdentitySelectionUpdated(selector CachedSelector, selections, added, deleted []identity.NumericIdentity) {
 	log.WithFields(logrus.Fields{
-		"selector":   selector,
-		"selections": selections,
-		"added":      added,
-		"deleted":    deleted,
+		logfields.EndpointSelector: selector,
+		logfields.PolicyID:         selections,
+		logfields.AddedPolicyID:    added,
+		logfields.DeletedPolicyID:  deleted,
 	}).Debug("identities selected by L4Filter updated")
+
+	// Skip updates on filter that wildcards L3 and/or L4.
```

Comment:
```suggestion
	// Skip updates on filter that wildcards L3.
```

This is only checking that L3 is wilcarded, right? Either `AllowsAllAtL3()` or there's no L3-dependent rules (meaning they're L7 rules with an L4 dependency only).

Reply:
Correct, fixed comment as suggested.